packages/devtools_app/lib/src/shared/console/widgets/display_provider.dart

```diff
@@ -15,6 +15,25 @@ import '../../diagnostics/dart_object_node.dart';
 import '../../primitives/simple_items.dart';
 import 'description.dart';
 
+VariableSelectionControls _selectionControls({
+  required DartObjectNode variable,
+  required Function(TextSelectionDelegate delegate)? onInspect,
```

Comment:
Nit: consider making this optional.

Reply:
I want it to be explicitly specified even when it is null.